### PR TITLE
[MM-19217] Include iPad in check for ios web

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

This PR makes it so a user visiting the webapp for the first time on an iPad will be prompted to download the app before continuing.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-19217

#### Additional Info

The `isIosWeb` function (which calls `isIosSafari`) is referenced in the root component (to check for app download), and in the `create_post` component. Its role in the `create_post` component is to hide the keyboard when screen orientation changes. I'm not aware of the ins and outs of this, or how the iPad being used here now affects the function.

https://github.com/mattermost/mattermost-webapp/blob/78156a7a22c9dfe75fe4636f87f411555fb2cdb2/components/create_post/create_post.jsx#L357-L363

### CODE REVIEW

File: utils/user_agent.jsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export function isSafari() {
 }
 
 export function isIosSafari() {
```
Comment (lieut-data) [inline utils/user_agent.jsx]: Just to check, will this mean that I can't load the desktop experience in iPad Safari? Or is it still smart enough to recognize the width and avoid showing the purely mobile view?

Comment (mickmister) [reply utils/user_agent.jsx]: This change should not affect how the app is presented. This function is only used to determine if we should show the "download app" page, and for the one case in `create_post` mentioned in the PR's description. The overall responsiveness of the app is unaffected.

**Comment by cpanato** [comment]
will look the error

**Comment by mickmister** [comment]
@cpanato I think it may be because I updated the branch mid-deploy

**Comment by cpanato** [comment]
lets see, also we are adding more clusters because the hackoctoberfest

**Pull request merged**